src/containerapp/azext_containerapp/custom.py

```diff
@@ -2370,6 +2374,57 @@ def delete_environment_telemetry_data_dog(cmd,
     return r
 
 
+def show_environment_telemetry_data_dog(cmd,
```

Comment:
For `show_environment_telemetry_otlp` , `show_environment_telemetry_data_dog`, `show_environment_telemetry_app_insights`, they all return following content, right?

```
{
    "enable-open-telemetry-metrics": false,
    "enable-open-telemetry-traces": false,
    "key": "Configured",
    "site": "datadoghq.com"
  }
```

Can these repeated codes organized into a method?